graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphCheckFactory.kt

```diff
@@ -6,7 +6,7 @@ import com.anaplan.engineering.azuki.core.system.UnsupportedCheck
 
 interface GraphCheckFactory : CheckFactory {
     fun hasVertexCount(graphName: String, count: Long): Check = UnsupportedCheck
-    fun hasShortestPath(graphName: String, from: Any, to: Any, shortestPath: List<Any>): Check = UnsupportedCheck
+    fun hasShortestPath(graphName: String, shortestPath: List<Any>): Check = UnsupportedCheck
```

Comment:
⛏️ I would call the argument `path` rather than `shortestPath`. The "shortest" part is kind of redundant and it doesn't _have_ to actually be a shortest path (in that case the check will just be false).